app/utils/themeUtils.js

```diff
@@ -3,15 +3,17 @@ import imageLogoDarkMode from 'app/assets/logo.png';
 import bannerLightMode from 'app/assets/om-abakus-banner.png';
 import bannerDarkMode from 'app/assets/om-abakus-banner-dark-mode.png';
 
+const isClientSide = () => typeof window !== 'undefined' && document;
+
 export const applySelectedTheme = theme => {
-  if (typeof window !== 'undefined') {
+  if (isClientSide()) {
```

Comment:
Just use `__CLIENT__` to avoid hardcoding this "hack". :smile:

```js
if (__CLIENT__) {
    [...]
```

instead.

See this for client:

https://github.com/webkom/lego-webapp/blob/611c13b6a6cb7b945a4af462f0cef47a25b01404/config/webpack.client.js#L84

and this for server

https://github.com/webkom/lego-webapp/blob/611c13b6a6cb7b945a4af462f0cef47a25b01404/config/webpack.server.js#L47

Reply:
Thanks for your feedback regarding this issue. A PR with the changes you suggested is now open.